update session 1 guidelines (#29)
theodo/dojo-serverless: docs/aws-console-guide.md

# AWS console guide

## CloudFormation: description of the whole infrastructure
![](../assets/CloudFormation_console_page.png)
When you search for `CloudFormation` in the search bar and click on the `dojo-serverless-backend-dev` pile you land on that page.
You have informations about the pile in the first tab `informations sur la pile`.
You can follow all update event of the resources you have in your formation in the second tab `événements`.

> All the resources that have been provide you deployed the infrastructure is listed in the `Ressources` tab.

Reviewer: ```suggestion
All the resources that have been provisionned in your infrastructure are listed in the `Ressources` tab.
```
